understanding/20/focus-order.html

```diff
@@ -124,17 +124,19 @@ <h2>Examples of Focus Order</h2>
             key expands a node, then using the down arrow key moves into the newly expanded nodes.
          </li>
          
-         <li>A web page implements modeless dialogs via scripting. When the trigger button is activated,
+         <li>A web page implements modeless dialogs. When the trigger button is activated,
             a dialog opens. The interactive elements in the dialog are inserted in the focus order
             immediately after the button. When the dialog is open, the focus order goes from the
             button to the elements of the dialog, then to the interactive element following the
             button. When the dialog is closed, the focus order goes from the button to the following
             element.
          </li>
          
-         <li>A web page implements modal dialogs via scripting. When the trigger button is activated,
+         <li>A web page implements modal dialogs. When the trigger button is activated,
             a dialog opens and focus is set within the dialog. As
-            long as the dialog is open, focus is limited to the elements of the dialog. When the
+            long as the dialog is open, all web page content outside the dialog becomes 
```

Comment:
Does this section say something slightly different to the [APG modal dialogs pattern keyboard interaction](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/#keyboardinteraction) section?

My personal view is that the APG needs to be changed to avoid focus being locked into the modal.